cxx-qt-build/src/lib.rs

```diff
@@ -258,7 +261,9 @@ impl CxxQtBuilder {
         Self {
             cpp_format: None,
             rust_sources: vec![],
-            qt_enabled: true,
+            qobject_headers: vec![],
+            qt_modules: vec!["Core".to_owned(), "Gui".to_owned()],
```

Comment:
If they can't have duplicates, then this should be a set rather than a vector ? like a std::collections::HashSet ?

Reply:
done